vrecord

```diff
@@ -308,7 +308,7 @@ duration_check(){
             _report -w "Illegal value for recording time. Input must only be numbers."
             exit 1
         fi
-        if [[ 1 -eq "$(echo "${duration} == 0" | bc)" ]] ; then
+        if [[ 1 -eq "$(echo "${duration} = 0" | bc)" ]] ; then
```

Comment:
duration="0.000" ; echo "${duration} = 0" | bc
(standard_in) 1: parse error

Reply:
Thank you, I’ll fix! (The behaviour is not clear to me from the top of the head.)

Reply:
see https://www.gnu.org/software/bc/manual/html_mono/bc.html#SEC11

Reply:
Ah, I understand, it’s not the double [[. – Thank you very much, as usual!